yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -185,41 +204,73 @@ func parseFleetConfigFile(filePath string) ([]AssessMigrationDBConfig, error) {
 	}
 	defer file.Close()
 
-	scanner := bufio.NewScanner(file)
-	for scanner.Scan() {
-		line := scanner.Text()
-		if len(line) == 0 {
-			continue
-		}
-		dbConfig := parseFleetConfigLine(line)
-		dbConfigs = append(dbConfigs, dbConfig)
+	reader := csv.NewReader(file)
+	header, err := reader.Read()
+	if err != nil {
+		return nil, fmt.Errorf("failed to read fleet config file header: %w", err)
 	}
+	header = normalizeFleetConfFileHeader(header)
+
+	lineNum := 2
+	for {
+		record, err := reader.Read()
+		if err == io.EOF {
+			break
+		} else if err != nil {
+			return nil, fmt.Errorf("failed to read line %d: %w", lineNum, err)
+		}
 
-	if err := scanner.Err(); err != nil {
-		return nil, err
+		if len(record) != len(header) {
```

Comment:
DO we need this here if we have already validated it?

Reply:
yeah no need, will remove it.